source/classification1.md

```diff
@@ -12,41 +12,26 @@ kernelspec:
   name: python3
 ---
 
-(classification)=
-# Classification I: training & predicting
-
 ```{code-cell} ipython3
 :tags: [remove-cell]
+import warnings
+def warn(*args, **kwargs):
+    pass
+warnings.warn = warn
```

Comment:
```suggestion
import warnings
warnings.filterwarnings("ignore", category=DeprecationWarning)
```

Does this work for you?

Reply:
I tried this originally, and it didn't work. Let me try it again now.

Reply:
ah. Duh. the trick is to ignore `FutureWarning`, not `DeprecationWarning`. Will edit this and ch6 for that